### ISSUE TITLE
Idea: Attract JavaScript front-end developers

### ISSUE THREAD

**Issue by dbooth-boston** [w3c/EasierRDF#30]
"front-end developers are a new generation.
They did not exist back when RDF was conceived. . . .
They’re using JavaScript, TypeScript, React, etc."
https://lists.w3.org/Archives/Public/semantic-web/2018Nov/0050.html

**Comment by joshuef** [comment]
For me, as a JS dev, there's a few key things that make RDF a pain. The main issue being HUUUGE amount of on-boarding needed. When I may only want to 'make some data'. There needs to be an easy way to ensure my data is RDF, without having to learn _everything_.

But there's decision paralysis with tooling, with vocabs / schema. Do I need to use SOLID? (and authentication thereon?) Can I not? etc etc.

For me, I think a few libs providing schema/vocab packages via `npm` would slim that down a lot. Couple with some auto-validation and documentation for forming your objects and all of a sudden making RDF can become just a bit more accessible. And then we can turn the conversation to 'why would you _not_ use RDF?'

And once they have that data, only _then_ do they need to start learning about the rest of the RDF landscape (IMO).

**Comment by RubenVerborgh** [comment]
Yes, I honestly think the name of this group is wrong. It shouldn't be "easy RDF", but "easy Linked Data". I don't think JS devs should be exposed to RDF—but they should be to Linked Data.

**Comment by dbooth-boston** [comment]
> I honestly think the name of this group is wrong. It shouldn't be "easy RDF", but "easy Linked Data".

@RubenVerborgh, I agree in hindsight, since the scope is not RDF itself, but the whole RDF ecosystem -- whose purpose is to make and work with Linked Data!  (And I mean "Linked Data" broadly here -- not limited to dereferenceable URIs, though those are nice.)
